Sources/GutenbergKit/Sources/EditorViewController.swift

```diff
@@ -36,8 +40,12 @@ public final class EditorViewController: UIViewController, GutenbergEditorContro
     private var cancellables: [AnyCancellable] = []
 
     /// Initalizes the editor with the initial content (Gutenberg).
-    public init(content: String = "", service: EditorService, siteApiRoot: String = "", siteApiNamespace: String = "", authHeader: String = "") {
+    public init(id: Int? = nil, type: String = "", title: String = "", content: String = "", service: EditorService, themeStyles: Bool = false, siteApiRoot: String = "", siteApiNamespace: String = "", authHeader: String = "") {
```

Comment:
I feel we are approaching an argument count threshold where we should consider an alternative pattern — builder pattern, parameter object, property setters, etc. We can address this separately, if desired.

Reply:
Yeah, I thought about it when I kept adding more 😅 I think we could do it separately so this PR doesn't get bigger than it already is.